app/src/DesignSystem.re

```diff
@@ -1,8 +1,12 @@
 open Css;
 
-module Operators = {
-  let (<<) = (a, b) => merge([a, b]);
-};
+let (<<) = (a, b) => merge([a, b]);
+
+let (<<?) = (a, b) =>
```

Comment:
With the `DesignSystem` growing it starts to make sense to introduce a `.rei` with documentation on what the different parts do and how to use them.

Reply:
That's one possibility.

I was thinking of doing the usage documentation on the living-styleguide end with examples where you can see how to use it and also how it's rendered. It might be harder to maintain if I have to keep the documentation in two places: style guide + `.rei`. What do you think?

Reply:
I think `.rei` files are non-avoidable and we should get in the habit of having them around by default. Do we want low-level code documentation in the styleguide? Thought of it more as a high-level showcase of what elements exist and how they look. Maybe there is a way to generate part of the styleguide from the `.rei` file, but haven't looked deeply into that part of reason yet.

Reply:
Added the `.rei` file + interface documentation.

Even though I think it's a good idea to have the interface documented, I think it might be a little too early to cement it in with comments and explicit typing.

I'm still figuring out what the interface should be and there's a lot of trial-and-error and re-writes when I find a better abstraction. I think at this stage it would be better to use the momentum to get as much elements done as possible, then refactor out the duplication and a bit later, when it all seems somewhat usable and doesn't change as much, add the docs/types.

On the other hand, maybe writing the docs upfront would make me think harder about the interface. I'll try to work on the `.rei` file as I go for now, perhaps we can re-evaluate the decision later if it's getting too much in the way.

Reply:
Wholeheartly agreed on all points, mostly trying to see if we can make it work.